libs/react-helpers/src/lib/links/links.ts

```diff
@@ -16,7 +16,7 @@ export const ExternalLinks = {
   GITHUB: 'https://github.com/vegaprotocol/token-frontend',
   DISCORD: 'https://vega.xyz/discord',
   GOVERNANCE_PAGE: 'https://vega.xyz/governance',
-  PROPOSALS_GUIDE: 'https://docs.vega.xyz/docs/mainnet/tutorials/proposals',
+  PROPOSALS_GUIDE: 'https://docs.vega.xyz/mainnet/tutorials/proposals',
```

Comment:
chore (blocking): this is hardcoded to mainnet, we should move this to the get docs link function aboe

Reply:
There's me being shown up as a lazy reviewer

Reply:
Good spot. I've now updated this and the proposal docs links.